### PR TITLE
Added Striped theme

### PR DESCRIPTION
**Pull request by gnotc**
Added Striped theme

### CODE REVIEW

**Comment by InsidiousMind** [comment]
Hey @digitalnomadstudio 

Your demo link seems to be broken, please fix this

**Comment by digitalnomadstudio** [comment]
Tis done!

**Comment by InsidiousMind** [comment]
@digitalnomadstudio are you sure you fixed it? this link: http://scribble.muan.co/2013/05/06/scribble-the-jekyll-theme/ still 404's

File: _posts/2017-05-05-Striped.markdown [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Striped
+date: 2017-05-05
+homepage: https://github.com/aroaminggeek/Jekyll-HTML5Up-Striped
+download: https://github.com/muan/scribble/archive/master.zip
+demo: http://scribble.muan.co/2013/05/06/scribble-the-jekyll-theme/
+author: A Roaming Geek
+thumbnail: striped.png
+license: CCA 3.0
```
Comment (gynter) [inline _posts/2017-05-05-Striped.markdown]: Please change license to it's proper name `Creative Commons Attribution 3.0 Unported License`

File: _posts/2017-05-05-Striped.markdown [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Striped
+date: 2017-05-05
```
Comment (gynter) [inline _posts/2017-05-05-Striped.markdown]: Please add time `00:00:00`.

**Pull request closed without merging**